api/app/signals/apps/feedback/exceptions.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework.exceptions import APIException
+from rest_framework.status import HTTP_410_GONE
+
+
+class Gone(APIException):
```

Comment:
Maybe this could be part of the core exceptions (api/generics?) instead of a specific feedback exception. I thought it already would have been part of DRF but apparently it is not.

Reply:
We then have `feedback` depending on `api` and `api` on `feedback`.

Reply:
I agree there may be a better spot. But currently `feedback` does not depend on `api` and `api` does depend on `feedback`. I don't want to make these dependencies going two ways.